boards/arm/mikroe_mini_m4_for_stm32/mikroe_mini_m4_for_stm32.dts

```diff
@@ -8,8 +8,8 @@
 #include <st/f4/stm32f415Rg.dtsi>
 
 / {
-	model = "STMicroelectronics Mikroe MINI-M4 for STM32 board";
-	compatible = "st,stm32f415-mikroe-f415rg", "st,stm32f415";
+	model = "Mikroe MINI-M4 for STM32 board";
+	compatible = "mikroe,mini-m4-for-stm32", "st,stm32f415rg";
```

Comment:
Why not just "mikroe,mini-m4-stm32"?

Reply:
The board's path is ```boards/arm/mikroe_mini_m4_for_stm32```. It seems that inside zephyr's buildsystem the board's name is ```mikroe_mini_m4_for_stm32```. It makes sense to use this name inside dts too.

Reply:
Compatible doesn't always follow board's path (e.g. stm32f*_disco boards).